pkg/util/strings/strings.go

```diff
@@ -115,3 +115,7 @@ func SplitAndTrim(s string, separator string) ([]string, error) {
 
 	return cleanSplit, nil
 }
+
+func ConvertSlashes(input string) string {
```

Comment:
Can we rename to specific like `ConvertToForwardSlash` ?

Reply:
This is really an existing function. I just moved it. Sure it can be renamed, but personally I don't think it is required. That said, there is actually a bug here which I think should be addressed in a follow up issue.

Reply:
Ok, another followup issue :)